### PR TITLE
Revert 738

### PR DESCRIPTION
**Pull request by makyo**
Reverts past work removing the deployer calls; instead, only removes the deployerImport calls.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2567/
Test PASSed.

**Comment by frankban** [comment]
:+1: thank you Madison!
To really restore the functionality, we need to re-enable the watchAll logic on logn, which was removed by @hatched in https://github.com/juju/juju-gui/commit/b964541c83d4511aee1ba02ef2ac0dd48c098622

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2616/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2617/
Test FAILed.

**Comment by makyo** [comment]
FAilure appears spurious

File: app/assets/javascripts/bundle-import-helpers.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('bundle-import-helpers', function(Y) {
```
Comment (hatched) [inline app/assets/javascripts/bundle-import-helpers.js]: As this is only for the messaging could we rename this module to something more representative?

**Comment by makyo** [comment]
# QA

Use juju-quickstart to deploy a bundle without using the uncommitted feature.  Do this while you have the gui deployed and open, make sure that you receive notifications.

**Comment by hatched** [comment]
+1 with naming change. Thanks for getting this back in!

**Comment by frankban** [comment]
:+1: QA ok.
I successfully run juju-quickstart with two or three committed bundles and I see notifications about started/scheduled bundle deployments.
Thank you for restoring this functionality!

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2618/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2619/
Test FAILed.

**Comment by makyo** [comment]
Thanks all :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/921

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2620/
Test PASSed.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**